.rubocop_todo.yml

```diff
@@ -1,52 +1,77 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 0`
-# on 2016-11-10 10:38:07 -0600 using RuboCop version 0.42.0.
+# on 2016-12-16 14:48:45 -0800 using RuboCop version 0.45.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 264
+# Offense count: 269
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 
-# Offense count: 1
+# Offense count: 2
+Lint/EmptyWhen:
+  Enabled: false
+
+# Offense count: 2
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, SupportedStyles.
 # SupportedStyles: runtime_error, standard_error
 Lint/InheritException:
   Enabled: false
 
-# Offense count: 21
+# Offense count: 20
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
-# Offense count: 33
+# Offense count: 31
 Lint/RescueException:
   Enabled: false
 
-# Offense count: 50
+# Offense count: 4
+# Cop supports --auto-correct.
+Lint/UnifiedInteger:
+  Enabled: false
+
+# Offense count: 2
+# Cop supports --auto-correct.
+Lint/UnneededSplatExpansion:
+  Enabled: false
+
+# Offense count: 48
 # Cop supports --auto-correct.
 # Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Manually added due to internal error in the rubocop rule that needs to be investigated further.
-# To repro and see stack trace, enable this and run: bundle exec rubocop bin/cron/deliver_poste_messages -d
-Lint/ShadowedException:
-  Enabled: false
```

Comment:
That was the one with the internal failure that was crashing rubocop. They must have fixed it.